src/main/resources/application.yml

```diff
@@ -14,4 +14,16 @@ management:
       jmx:
         exposure.include: "*"
       web:
-        exposure.include: info, health
\ No newline at end of file
+        exposure.include: info, health
+app:
+    pdnd:
+      base-url: "\${PDND_BASE_URL:https://auth.uat.interop.pagopa.it}"
+      config:
+        audience: "\${PDND_ACCESS_TOKEN_AUDIENCE:auth.uat.interop.pagopa.it/client-assertion}"
+        privateKey: "\${PDND_PRIVATE_KEY:}"
+        publicKey: "\${PDND_PUBLIC_KEY:}"
+      anpr:
+        config:
+          client-id: "\${PAGOPA_PDND_CONFIGURATION_CLIENT_ID:890b7ca9-b402-4dce-9e8d-9a333d22d76d}"
+          kid: "\${PAGOPA_PDND_CONFIGURATION_KID:jxOpPRxM6oFcnnKtICqeW5l7fbxLr45IAsJ8Q9s-fK8}"
```

Comment:
clientId and kid could be the same for all the integrated services, but it could be more flexible having the possibility to configure them for each service.
Anyway, kid and private/public keys should be configured on the same level (or both pdnd.config or both on pdnd.anpr.config):
lets move privateKey and publicKey under pdnd.anpr.config